modules/aerodyn/src/AeroAcoustics_Types.f90

```diff
@@ -73,7 +73,7 @@ MODULE AeroAcoustics_Types
 ! =======================
 ! =========  AA_InputFile  =======
   TYPE, PUBLIC :: AA_InputFile
-    REAL(DbKi)  :: DT_AA      !< Time interval for aerodynamic calculations {or "default"} [s]
+    REAL(DbKi)  :: DT_AA      !< Time interval for aerodynamic calculations {or  [default"}"]
```

Comment:
Units missing.  Probably the double quotes?

Reply:
Resolved in #1596